lib/assessment.js

```diff
@@ -8,23 +8,27 @@
 // Question 1
 // Write a callback for reduce that selects
 // the minimum value in the array
-var min = function min() {
-
+var min = function min(previousValue, currentValue) {
+  return previousValue < currentValue? previousValue : currentValue;
 };
 
+
 // Question 2
 // Write a callback for reduce that produces
 // the sum of the values in the array
-var sum = function sum() {
-
+var sum = function sum(previousValue, currentValue) {
+  return previousValue += currentValue;
 };
 
+
 // Question 3
 // Write a function that returns true
 // if invoking the callback for any element
 // in the array returns a truthy value
 var any = function any(array, callback, thisArg) {
-
+  if (array.every(callback) === true){
+    return true;
+  }
```

Comment:
Hah! Clever, but no.

Think back to the examples above. Is there a way to test if any of the elements pack a particular test? What if we check each of their results?